app/angular-master-class.html

```diff
@@ -169,6 +171,17 @@ <h2 class="thtrm-m-section__sub-headline">Server Communication</h2>
                   </ul>
                 </div>
               </div>
+              <div class="thtrm-l-grid-u--1-1 thtrm-l-grid-u--medium-1-2">
+                <div class="thtrm-m-content-box">
+                  <h2 class="thtrm-m-section__sub-headline">Upgrading to Angular 2</h2>
+                  <ul class="thtrm-m-list thtrm-m-list--checked">
+                    <li>Upgrading strategies</li>
+                    <li>Preparing the upgrade</li>
+                    <li>Using Angular 2 components in Angular 1</li>
+                    <li>Using Angular 1 components in Angular 2</li>
```

Comment:
How about "Making Angular 1 and Angular 2 components interoperable"

We should also add sth. about services/providers